Gems/Atom/RHI/Code/Platform/Android/renderdoc_support_android.cmake

```diff
@@ -6,5 +6,4 @@
 #
 #
 
-set(ATOM_RHI_TRAIT_BUILD_SUPPORTS_TEST FALSE)
-set(ATOM_RHI_TRAIT_BUILD_SUPPORTS_EDIT FALSE)
+set(PAL_TRAIT_BUILD_RENDERDOC_SUPPORTED FALSE)
```

Comment:
Probably not a huge deal but I thought we supported RenderDoc on Android

Reply:
there wasn't any file to enable it there

Reply:
I wasn't sure about this myself and didn't have an Android to test with unfortunately. My understanding is that RenderDoc may actually ship as part of the standard Android Studio toolchain, but I think it's ok to leave this as is and possibly have @AMZN-tpeng verify what the correct RenderDoc-on-Android workflow is later